lib/diagnostic.rb

```diff
@@ -107,9 +135,16 @@ def eat_meat(food)
 # Then, create a new class method called "get_possible_moves",
 # which returns the string "kick, move, punch"
 
+## Question 7 is not complete I was in the middle of analyzing it,
+# I did create a class method, but haven't fixed the ComboAttack class.
+
 class ComboAttack
   attr_reader :moves, :damage
 
+  def self.get_possible_moves
+    "kick, move, punch"
+  end
```

Comment:
The way to make the method chaining work is for `punch`, `kick`, and `move` to return `self`.That way, `ComboAttack.new.punch.move('left').kick.damage` is equivalent to

```ruby
combo = ComboAttack.new
combo.punch
combo.move('left')
combo.kick
combo.damage
```